backend/backend/urls.py

```diff
@@ -43,6 +44,7 @@
                     path("api-token-auth/", obtain_auth_token),  # for expiry token authent
                     path("api-token/", obtain_auth_token_already_authenticated),
                     path("active-api-tokens/", active_bearer_tokens),
+                    path("unactivated-users/", UserAwaitingApprovalViewSet.as_view({"get": "list"})),
```

Comment:
Why is this needed?

Reply:
I assume you mean .as_view
This is because I made a ViewSet instead of a simple View
I'm not sure why this was at the start of this feature I need to check but I feel like I didn't really have a choice

Reply:
I mean the `({"get": "list"}))`